projects/br_scs/test/Mirza/BusinessRegistry/Tests/Dummies.hs

```diff
@@ -4,21 +4,17 @@ module Mirza.BusinessRegistry.Tests.Dummies where
 
 import           Data.GS1.EPC                 (GS1CompanyPrefix (..))
 import           Mirza.BusinessRegistry.Types as BRT hiding (businessName)
-import           Mirza.Common.Types           as CT
 import           Mirza.Common.Utils
 
-import           Mirza.Common.Tests.Utils     (unsafeMkEmailAddress)
-
 import           Data.Text
 
 dummyBusiness :: BRT.NewBusiness
 dummyBusiness = NewBusiness (GS1CompanyPrefix "3000001") businessName (mockURI businessName) where
                 businessName = "pubKeyTests_businessName"
 
 dummyNewUser :: BRT.NewUser
-dummyNewUser = makeDummyNewUser (unsafeMkEmailAddress "fake@gmail.com")
+dummyNewUser = makeDummyNewUser "fake@example.com"
```

Comment:
We don't actually have any tests for the JWT auth, do we? This should make the tests fail. I will try to add tests soon.

Reply:
Yeah sorry I should have been more clear on that in PR, this higher level feature is effectively being done over a number of PR's, fixing the tests and developing a JWT test workflow will be in a subsequent PR that I make. This list of additional items is here: https://github.com/data61/Mirza/issues/385#issuecomment-483510242